Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -151,10 +157,26 @@ open class MessageContentCell: MessageCollectionViewCell {
         let topMessageLabelText = dataSource.messageTopLabelAttributedText(for: message, at: indexPath)
         let bottomMessageLabelText = dataSource.messageBottomLabelAttributedText(for: message, at: indexPath)
 
+        let sentDate = message.sentDate
+        let sentDateString = MessageKitDateFormatter.shared.string(from: sentDate)
```

Comment:
I think we should offer the same level of customisation for this as we offer for other cell labels.